### PR TITLE
Use Fira Code for more consistent monospace font styling

### PR DESCRIPTION
**Pull request by rehandalal**
r?

### CODE REVIEW

File: extension/content/less/cm-ndt.less [hunk 1]
```diff
@@ -44,6 +44,8 @@
     border: 1px solid var(--cm-border);
     border-radius: @input-border-radius;
     color: var(--cm-color);
+    font-family: @font-family-monospace;
+    font-variant-ligatures: no-contextual;
```
Comment (mythmon) [inline extension/content/less/cm-ndt.less:48]: Not a fan of ligatures?

**Review by mythmon — APPROVED** [review]
bors merge

File: extension/content/less/cm-ndt.less (hunk 1 above)
Comment (rehandalal) [reply extension/content/less/cm-ndt.less:48]: I am but I think they should be optional with "off" being the default.

Comment (rehandalal) [reply extension/content/less/cm-ndt.less:48]: I'll be adding UI for extension settings that will let us customize some of the appearance stuff like this.

**Pull request merged**